UnityProject/Assets/Scripts/UI/Core/Net/Elements/NetButton.cs

```diff
@@ -10,11 +10,20 @@ namespace UI.Core.NetUI
 	[Serializable]
 	public class NetButton : NetUIStringElement
 	{
+
+		private Button Button;
+
 		public UnityEvent ServerMethod;
 
 		public override void ExecuteServer(PlayerInfo subject)
 		{
 			ServerMethod.Invoke();
 		}
+
+		public void Awake()
+		{
+			Button = this.GetComponent<Button>();
```

Comment:
would be cool if you add a null check to this just in case

Reply:
or make it so that this component automatically adds the button component if its added in the inspector 
```cs
[RequireComponent(typeof (Button))]
```

Reply:
yeha it has it
namespace UI.Core.NetUI
{
	/// Simple button, has no special value
[RequireComponent(typeof(Button))]